docs/wxm-token.md

```diff
@@ -20,7 +20,7 @@ the people that have contributed to the Network development as well as people wh
 | Allocation    | Usage 		|  Details |
 | -------------: | :----------- | :---- |
 | 52m   | Station rewards 	| 14,246 tokens will be distributed to station owners every day for providing data to the Network.|
-| 3m    | Beta testers|   3m tokens will be distributed to station owners that participated in the Network during its beta phase. |
+| 3m    | Beta testers|   3m tokens [will be distributed](/docs/station-rewards.html#beta-rewards) to station owners that participated in the Network during its beta phase. |
```

Comment:
Same here, make `distributed` to `allocated`.

Reply:
I will leave it here. This is a high level view, more plain English and less ETH. These tokens are intended to be distributed to station owners.